guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Upgrading_the_External_Database_{context}"]
+= Upgrading the External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+* Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
+ifdef::satellite[]
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersion}/html/installing_satellite_server_in_a_connected_network_environment/performing-additional-configuration#using-external-databases_satellite[Using External Databases with {Project}].
+endif::[]
+ifdef::katello[]
+For more information, see https://docs.theforeman.org/{ProjectVersion}/Installing_Server/index-katello.html#using-external-databases_foreman[Using External Databases with {Project}].
+endif::[]
```

Comment:
This can be written using macros in one go:
```suggestion
ifdef::katello,orcharhino,satellite[]
For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
endif::[]
```

(we do not include external DB for foreman builds, no idea why

Reply:
> (we do not include external DB for foreman builds, no idea why

8710bde2c7185c057d7f9a71313d7c166dae5fda. When I was reading through it, it only worked if you had Katello. If we add more conditionals to hide the Katello-specifics (like candlepin & pulp's databases) it can be included.